autocertkit/utils.py

```diff
@@ -65,6 +65,7 @@
 
 # Capability Tags
 REQ_CAP = "REQ"
+MULTICAST_CAP = "VM_MULTICAST"
```

Comment:
The capability should be 'MULTICAST' - we're testing the hardware, and there's nothing special about the VM.